src/validations/admins.js

```diff
@@ -1,5 +1,40 @@
 const Joi = require('joi');
 
+const validateUpdate = (req, res, next) => {
+  const adminValidation = Joi.object({
+    firstName: Joi.string()
+      .min(3)
+      .max(15),
+    lastName: Joi.string()
+      .min(3)
+      .max(15),
+    dni: Joi.number()
+      .positive()
+      .min(10000000)
+      .max(99999999),
+    phone: Joi.number()
```

Comment:
Phone must be a string. Keep in mind that min() and max() value will be changed with string length (9 and 12) and you have to delete positive()